game.js

```diff
@@ -50,14 +50,15 @@ const inter = setInterval(() => {
     timeLeft--
```

Comment:
This comment is not for this line, the silly PR won't let me comment the actual line I need.

Line 47 where you declare timeLeft has a missing semi colon!